honeybadger-client/src/main/java/com/workable/honeybadger/HoneybadgerClient.java

```diff
@@ -116,9 +128,10 @@ public HoneybadgerClient(String apiKey) {
      *                                 dispatched
      * @param excludedExceptionClasses Comma delimited list of Exceptions that should be ignored
      */
-    public HoneybadgerClient(String apiKey, String excludedSysProps, String excludedExceptionClasses) {
```

Comment:
I would overload this method, so someone that updates the library version does not need to update also the code, if he does not care about excludedExceptionCauses. So I would leave the old constructor and add this as a new one.

Reply:
👍

Reply:
Done